<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gesformation</class>
 <widget class="QMainWindow" name="gesformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gesformation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="rechercher">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>rechercher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trier">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>trier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editidmodif">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>410</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="liste">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>851</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendrier">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="formulaire">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>Formulaire de la formation</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="nom">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLineEdit" name="editcertif"/>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="prix">
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLineEdit" name="editorganisme"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="statut">
       <property name="text">
        <string>Statut</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="fin">
       <property name="text">
        <string>Date de la fin</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="certif">
       <property name="text">
        <string>Validité du certificat</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="description">
       <property name="text">
        <string>description</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLineEdit" name="editprix"/>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="editdebut"/>
     </item>
     <item row="12" column="0">
      <widget class="QLineEdit" name="editstatut"/>
     </item>
     <item row="18" column="1">
      <widget class="QPushButton" name="ajouter">
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="id">
       <property name="text">
        <string>id</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="organisme">
       <property name="text">
        <string>Organisme</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLineEdit" name="editdescription"/>
     </item>
     <item row="8" column="0">
      <widget class="QLineEdit" name="editfin"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="editnom"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="debut">
       <property name="text">
        <string>Date du début</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="editID"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="exporter">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>exporter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifier">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>530</y>
      <width>93</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editnomattribut">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>450</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="listeC">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rappel">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>350</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>rappel</string>
    </property>
   </widget>
   <widget class="QLabel" name="nom_attribut">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>430</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nom_attribut</string>
    </property>
   </widget>
   <widget class="QTableView" name="listeR">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomrech">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>470</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="idmodif">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>390</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editnouvval">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>500</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="idsupp">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>320</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_stats">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>stats</string>
    </property>
   </widget>
   <widget class="QLabel" name="nouvval">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>480</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nou_val</string>
    </property>
   </widget>
   <widget class="QLabel" name="arduino">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>580</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>arduino</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idarduino">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>600</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>630</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>envoyer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1153</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
